crunchy-postgres/charts/crunchy-postgres/values.yaml

```diff
@@ -6,13 +6,15 @@ postgresVersion: 15
 #postGISVersion: '3.3' # use this version of POSTGIS. both crunchyImage and this property needs to have valid values for POSTGIS to be enabled.
 imagePullPolicy: IfNotPresent
 
+clusterName: wfprev-#{ENV}#
```

Comment:
Should this be named something like pituw instead of wfprev? Thinking about the future, where we'll presumably be storing backups for Claims Calculator in the same S3 bucket, so for those we might name it pitcc or pitclaim to differentiate.